openrtb-core/src/main/protobuf/openrtb.proto

```diff
@@ -1457,6 +1536,20 @@ message BidResponse {
       // Required for Flex Ads.
       optional int32 hratio = 27;
 
+      // List of supported APIs for the markup. If an API is not explicitly
+      // listed, it is assumed to be unsupported.
+      // Ignored by Google.
+      repeated APIFramework apis = 31 [packed = true];
```

Comment:
According to AC, this should be a part of the Banner object. But the spec says that it should be in the Bid object. It looks like there's an error in the AC.

Reply:
I updated the AC on the ticket to say it should be the BidResponse.seatbid.bid object 👍